src/corner_kick/scripts/start_test.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 # Start test script for Corner Kick.
 
+PERCY_TOKEN=369660795fa79afd24f25e21a4f8803e9d56beae94a31fbd4024ad34cf1c8cd1
```

Comment:
In general I think we should store private tokens like this in the travis ci config. I'm happy to move it over or to point you to where it should go if that's cool?

Reply:
Resolved since Percy is no longer in scope.